index.test.js

```diff
@@ -1,6 +1,6 @@
 import assert from 'node:assert'
-import { test, afterEach } from 'node:test'
-import * as base32 from 'thirty-two'
+import { test } from 'node:test'
+import base32 from 'thirty-two'
```

Comment:
@kentcdodds can you confirm this works for you locally? I found that it fixed some type errors in my project when I imported it this way. Happy to revert if it breaks on your side

Reply:
If it works for you and CI then it's good enough for me :)